src/wallet/sqlite.cpp

```diff
@@ -472,7 +509,19 @@ bool ExistsSQLiteDatabase(const fs::path& path)
 
 std::unique_ptr<SQLiteDatabase> MakeSQLiteDatabase(const fs::path& path, const DatabaseOptions& options, DatabaseStatus& status, bilingual_str& error)
 {
-    return MakeUnique<SQLiteDatabase>(path, path / DATABASE_FILENAME);
+    fs::path file = path / DATABASE_FILENAME;
```

Comment:
c3aeda3d2f7d7b34b8a35ccea478161b0ca2c80d, nit
```suggestion
    const fs::path file = path / DATABASE_FILENAME;
```

Reply:
Done

Reply:
Done